ci-operator/step-registry/openstack/provision/etcd-on-local-disk/openstack-provision-etcd-on-local-disk-commands.sh

```diff
@@ -24,6 +24,26 @@ function wait_for_machineconfig_done() {
   oc wait clusteroperators --timeout=30m --all --for=condition=Progressing=false
 }
 
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+oc_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
```

Comment:
nit: this is the openshift or cluster version, not the version of `oc`.

```suggestion
openshift_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
```